src/lifetracker/logic/LogicImpl.java

```diff
@@ -14,11 +13,13 @@
 
     private static final String ERROR_SAVE = "Warning: There was an error saving to the save file!";
     private static final String ERROR_INVALID_COMMAND = "Error: Command was not a valid command!";
+    private static final String COMMENT_SAVE = "Calendar is saved at ";
 
     private Parser commandParser;
     private Storage calendarStorage;
     private CalendarList calendar;
     private Stack<CommandObject> commandStack;
+    private StorageAdapter storageAdapter;
```

Comment:
No need for an interface/implementation pair here. StorageAdapter is used withing your own package only.